src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java

```diff
@@ -413,18 +412,12 @@ private void checkExcludedDates(Appeal appeal) {
     }
 
     private void checkMobileNumber(Contact contact, String personType) {
-        if (contact != null && contact.getMobile() != null && !isPhoneOrMobileNumberValid(contact.getMobile())) {
-            warnings.add(getMessageByCallbackType(callbackType, personType, getWarningMessageName(personType, null) + MOBILE, IS_INVALID));
+        if (contact != null && contact.getMobile() != null && !isMobileNumberValid(contact.getMobile())) {
+            errors.add(getMessageByCallbackType(callbackType, personType, getWarningMessageName(personType, null) + MOBILE, IS_INVALID));
```

Comment:
Why its change to Error instead of Warning? are we not giving change to case worker to correct it?

Reply:
It breaks the CCD definition regex if we allow the case to be created as an incomplete case and it ends up throwing an exception. They can still correct it by pressing the 'Previous' button.